### ISSUE TITLE
[a11y]: Tearsheet previous and next buttons are not screen reader accessible

### ISSUE THREAD

**Issue by elycheea** [carbon-design-system/ibm-products#5738]
### Package

@carbon/ibm-products

### Browser

Firefox

### Operating System

Windows

### Package version

v2.44.0-rc.1

### Automated testing tool and ruleset

manual

### Assistive technology

JAWS

### Description

Reported by Raghu during screen reader testing.

This refers to previous and next buttons (visually presented as  `<`, `>` symbols).

The prev and next buttons do not receive tab focus.
Also controls are not identified in <kbd>Arrow</kbd> key navigation, completely inaccessible for screen reader users.

> [!NOTE]
> Developers were not able to see this Next and Previous elements. Need to check with Raghu and team for screenshot when ever we were able to see these elements.

### Recommended fix

Confirm the functional purpose of these controls:
- Make the controls keyboard reachable/operable
- Make sure there are proper label, role, and state assigned to these controls

### User impact
When the screen reader doesn’t identify all page elements, it impacts the experience of users who are blind or have vision impairment. They can miss important information or fail to perform an action on the web page.

### WCAG 2.1 Violation

4.1.2. Name Role Value

### Reproduction/example

https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-tearsheet--with-navigation&globals=viewport:basic

### Steps to reproduce

Reported in the [Tearsheet with navigation](https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-tearsheet--with-navigation&globals=viewport:basic) story.

Navigate tearsheet using arrow keys.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems

**Comment by makafsal** [comment]
@elycheea @matthewgallo The previous (`<`) and next (`>`) buttons are absolutely hidden from UI and Screen readers as well.


![Image](https://github.com/user-attachments/assets/34c54196-963c-42c2-a8fe-f0619c3a568f)

It is only visible if the tab list can't contain items within the given width:


![Image](https://github.com/user-attachments/assets/856f818c-0ad2-463e-a2dd-531734a606ec)

`aria-hidden=true` and `tabindex=-1` are there.

![Image](https://github.com/user-attachments/assets/b0ae83dd-39d3-4265-a2f7-dc0e321dc612)

In addition to that, it has a `display: none` style


![Image](https://github.com/user-attachments/assets/a9e221f1-e65f-4229-b9f4-5c2bfbbf0162)

He mentions his friend can see it in their system visually, possibly because of their screen resolution or zoom level.

**Comment by makafsal** [comment]
@elycheea @matthewgallo These buttons are from the Carbon `Tabs` component, it has a `tabindex=-1`, which is why it is not accessible using the Tab key


![Image](https://github.com/user-attachments/assets/7e30109f-0435-4000-a401-332fc7472552)

In the Carbon docs, it is mentioned that:


[![Image](https://github.com/user-attachments/assets/f24eb2b1-a207-4e84-98c2-39e4af5180f8)](https://carbondesignsystem.com/components/tabs/accessibility#accessibility-testing-status)

**Comment by elycheea** [comment]
@makafsal Can we check if there is an equivalent issue in the Carbon core repository or if it’s an issue in our implementation? It sounds like this _is_ the expected behavior, but if this is being flagged for us both not in Carbon core, we may need to make adjustments.

**Comment by davidmenendez** [comment]
confirmed that this is the intended behavior for the carbon [Tabs component](https://react.carbondesignsystem.com/?path=/story/components-tabs--default). once a `Tab` is in focus via the tab key the other tabs are accessible via arrow keys. this is the correct behavior seen in `Tearsheet` and in the example of `tearsheet with navigation` the next tab click appropriately taking you to the next input.

if this behavior is incorrect it needs to be addressed in core. we should follow up with Raghu to make sure this is correct.

**Comment by makafsal** [comment]
@davidmenendez The `left` and `right` keys achieve functionality for keyboard users. It satisfies Raghu's concern.

**Comment by makafsal** [comment]
As per the discussion with Regu, this is the expected behaviour, hence closing this issue.
